scripts/utils.ps1

```diff
@@ -46,14 +46,23 @@ function Write-Header([string]$text) {
 function Get-ExecutablePath([string]$name, [string]$directory, [string]$envVar) {
     $path = [environment]::GetEnvironmentVariable($envVar, "Process")
 
-    if (!$path) {
-        if (!$directory) {
-            $path = Exec { & where.exe $name } | Select-Object -First 1
-        }
-        else {
-            $path = Exec { & where.exe /R $directory $name } | Select-Object -First 1
+    try
+    {
```

Comment:
Also we do use the java notation because the powershell plugin for vscode auto-format with this behavior.

Reply:
OK